### ISSUE TITLE
Can I use hand only data to train the network?

### ISSUE THREAD

**Issue by aragakiyui611** [IDEA-Research/OSX#46]
Can I load a full body pretrained weight and train the model on a hand only dataset like InterHand26M? Thank you every much!

**Comment by linjing7** [comment]
Hi, thanks for your question. Training the model on the hand-only dataset will definitely improve the accuracy of hand mesh recovery. You can try:
1) Crop-paster augmentation, i.e., crop out the original hand image from the whole-body image and use the hand-only image to replace it. You need to align the wrist joint in the whole-body image with the root joint in the hand-only image.
2) Mix the hand-only dataset with the whole-body dataset, and then OSX directly. You need to set the validity of the body keypoints and face keypoints to zero when computing the loss.

**Comment by aragakiyui611** [comment]
Thank you for your patient answer！
